scripts/common.sh

```diff
@@ -10,4 +10,25 @@ log(){
   else
     echo "$(date '+%Y-%m-%d %H:%M:%S') [INFO] $1"
   fi
+}
+
+# Fetch the value of requested variable defined in config file
+# params: $1 - variable name, $2 - config file full path
+get_variable(){
+  if [ "$#" -eq 2 ]; then
+    if [ ! -f "${2}" ]; then
+      echo "File ${2} is not existed."
+      return 1
+    fi
+    local VALUE=$(grep -o '^[^#]*' "${2}" | grep "${1}" | sed 's/ //g' | grep "${1}=" | sed -nE 's/^.*"(.*)".*$/\1/p')
+    if [ ! $(echo "${VALUE}" | wc -l) -eq 1 ];then
+      echo "ERROR - '${1}' is re-defined in '${2}'";
+      echo "${VALUE}"
```

Comment:
Does this print both values?

Reply:
yes, to show the multiple definition, but I think is not needed, is enough to let the user know the variable is multiple defined. 

I removed this line.